docs/home.md

```diff
@@ -48,6 +48,30 @@ Use the Classic Upload Build Step to upload an ant-style glob of files (File Pat
 * **Show inline in browser** to set the metadata of the files such that the file is shown inline in browser, rather than downloaded.
 * **Strip path prefix** and specify the prefix you want to strip if you don’t want the whole path of the local files reflected in the bucket object name.
 	
+### Pipeline Example
```

Comment:
s/Pipeline Example/Jenkins Declarative Pipeline
Add pre-cursor steps similar to: https://github.com/jenkinsci/google-kubernetes-engine-plugin/blob/develop/docs/Home.md#jenkins-declarative-pipeline

Also, please include examples of both each of the steps supported for use in the DSL.

Reply:
Just noticed you do include the other steps.